src/ppx/reactjs_jsx_ppx_v2.ml

```diff
@@ -73,12 +73,12 @@ let transformChildrenIfList ~loc ~mapper theList =
     (* not in the sense of converting a list to an array; convert the AST
        reprensentation of a list to the AST reprensentation of an array *)
     match theList with
-    | {pexp_desc = Pexp_construct ({txt = Lident "[]"}, None)} ->
+    | {pexp_desc = Pexp_construct ({txt = Lident "[]"; _}, None); _} ->
```

Comment:
I think this warning is way too sensitive. You can customize the flags to put the `_` back.